lib/index.ts

```diff
@@ -393,29 +393,12 @@ export default class Backburner {
     let method;
     let target;
     let methodOrTarget;
-    let methodOrWait;
     let methodOrArgs;
 
     if (length === 0) {
       return;
     } else if (length === 1) {
       method = args.shift();
-    } else if (length === 2) {
```

Comment:
is this a safe change? Seems like it is breaking no?

Reply:
I would love to make this change, I really dislike the complexity it aims to remove.

Reply:
nope it is not breaking, if you look `last else block` already handles `2+` arguments length and there is test coverage already for it when `bb.later(()=>{}, 123)`

Reply:
lets shipit then